### DISCUSSION TITLE
[Funding] Funding for FOSS projects/functions

### DISCUSSION THREAD

**Discussion by machiav3lli** [FossifyOrg/General-Discussion#395] [Ideas]
**Checklist**
 - [x] I made sure that there are no existing issues - open or closed - to which I could contribute my information.
 - [x]  I have read the FAQ and my problem isn't listed.
 - [x]  I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
 - []  This issue contains only one feature request.
 - [x]  I have read and understood the [contribution guidelines.](https://github.com/FossifyOrg/General-Discussion?tab=readme-ov-file#faq)

**Is your feature request related to a problem? Please describe.**
There are for now no FOSS rich-photo-editing, nor integration of LLMs in FOSS gallery apps etc..

**Describe the solution you'd like**

Building upon the legacy of simple, but being more free from the word "simple". It'd maybe make sense to open the project for bigger funded contributions. One of the possibilities is the [Prototype Fund](https://prototypefund.de/en/) for Germany-based individuals/teams.

**Describe alternatives you've considered**

There are surely more features and other sources for funding that fits in this schema, so I'd call maybe for a crowd-sourcing, maybe this would interest some devs with enough capacity.

**Comment by inson1** [comment]
so the best would be to create rich-photo-editing library as separate project, that would be the best way right?
so other projects can use it as well, with this goal, it should be easier to win the funding

**Reply by machiav3lli** [reply]
Depends: theoretically, this would be the more "open" approach, but this also depends on use cases and willing of "others" to participate. I think this function is one of the last show-stoppers for an all-FOSS Android Ecosystem, but how to reach and activate specific expertise and capacities is the challenge here… this is why I see this project (FOSSify Gallery) as an initial core for such planning…

You can also count me in in planning and contribution, as soon as the Neo Store and Neo Launcher v1s are stable.
